README.md

```diff
@@ -416,6 +416,66 @@ The local key helper is responsible for:
 - Producing signatures with the binding key
 - Cleanup of the binding key and its artifacts (when the user clears the browser session or the key is unused for a long time)
 
+From the deployment point of view there are two types of local key helpers: _well-known_ and _third party_.
+
+_Well-known local key helpers_ come with either OS or built into the browser. It assumed that well-known key helpers are **trusted and enabled by default** in a browser. A browser knows how to activate a well-known key helper.
+
+A third party local key helper has a special deployment mechanism.
+
+### Local key helper on Windows
+
+On Windows, a Local key helper is a COM class. A COM interface that the local key helper implements is TBD (working name for this document is ILocalKeyHelper)
+
+#### Deployment of a 3rd party local key helper.
+
+Local key helpers will be deployed using registry.
+
+The base registry key is:
+
+```````
+[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\<Company>\<Browser>\LocalKeyHelperList]
+```````
+i.e. 
+
+```````
+[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge\LocalKeyHelperList]
+
+[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Google\Chrome\LocalKeyHelperList]
+```````
+
+Every local key helper has a child key in the above registry key. The entry for the local key is a registry key with the name equals to the local key helper id, the default value stores activation ID, prefixed by an activation scheme.
```

Comment:
Just thinking - with this proposed mechanism of registering helperIds and IDP declaring which helpers to use, do we still need an additional policy? I think we don't need it anymore, but wanted to confirm your thinking

Reply:
If Google ok with removing policy, i'm ok.